### ISSUE TITLE
Google Now feed integration does not works

### ISSUE THREAD

**Issue by mark-szabo** [Deletescape-Media/Lawnchair#1098]
Cannot swipe right to access Google Now feed nor does the Google button works on my OnePlus 5T.

## Description
Installed from Play Store, Google Now feed does not seems to work. Moreover when I click on the Google button top left nothing happens.

## Expected Behavior
Open Google Now feed? 😁

## Actual Behavior
Nothing. 

## Steps to Reproduce
1. Install from store.

## Environment
* Device: OnePlus 5T (A5010)
* Android version: 7.1.1
* Launcher version: 1.1.0.1742
* Rom: Stock ROM (OxygenOS 4.7.4)

**Comment by msfjarvis** [comment]
Did you install lawnfeed?

**Comment by mark-szabo** [comment]
Nope. Thanks! Maybe you should add a link in the app settings to download it from there.

**Comment by divadsn** [comment]
We are not allowed and therefore we had to revert that with #981.

**Comment by mark-szabo** [comment]
Ohh, I see. Sad to hear. 😔 
Thanks for the heads-up!

**Comment by demeterandras** [comment]
I installed Lawnfeed (also just found out about it) but swiping right still does nothing. Google Now feed is enabled in Lawnchair settings. Same behaviour as mark-szabo described before: no feed, no searchbar activity when tapping on Google icon or searchbar. Tried everything: disable-enable feed from settings, resetting Lawnchair, reinstalling Lawnchair+Lawnfeed, no difference.

Environment:
Device: Honor 8 Lite
Android version: 7.0
Launcher version: 1.1.0.1742
Rom: Stock ROM (EMUI 5.0.1), version PRA-AL00C00B211
